src/index.ts

```diff
@@ -0,0 +1,17 @@
+import HexColorPicker from "./packages/hex";
+import HexInput from "./packages/HexInput";
+import HslColorPicker from "./packages/hsl";
+import HslStringColorPicker from "./packages/hslString";
+import HsvColorPicker from "./packages/hsv";
+import RgbColorPicker from "./packages/rgb";
+import RgbStringColorPicker from "./packages/rgbString";
```

Comment:
**Discussion: file names and structure:**

`packages` folder name seems outdated since we ship one tree-shakable package.
`hex.tsx` doesn't describe the file as a component code. 

My offer:
```
/src
  index.tsx
  
  /components
    HexColorPicker.tsx
    RgbColorPicker.tsx
    ...

    /common
      ColorPicker.tsx
      HexInput.tsx
      Hue.tsx
      ...
  
```

Reply:
Yeah, I agree, that's nicer.

Reply:
Done